src/mono/wasm/Wasm.Build.Tests/TestMainJsProjectProvider.cs

```diff
@@ -22,8 +22,6 @@ public TestMainJsProjectProvider(ITestOutputHelper _testOutput, string? _project
     protected override IReadOnlyDictionary<string, bool> GetAllKnownDotnetFilesToFingerprintMap(AssertBundleOptionsBase assertOptions)
         => new SortedDictionary<string, bool>()
             {
-               { "dotnet.js", false },
```

Comment:
@radical I have removed `dotnet.js` from boot json, but it is expected to be present in the _framework. How would you this method to be updated?

Reply:
I have added a hotfix to unblock WBT in this PR